package-lock.json

```diff
@@ -5287,4 +5287,4 @@
       }
     }
   }
-}
+}
```

Comment:
Please don't make any changes in package-lock.json

Reply:
appologies for that, pretier extension just make it don this